testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt

```diff
@@ -173,8 +173,9 @@ class MiniAppDisplayActivity : BaseActivity() {
                 )
             }
         }
+        //miniAppMessageBridge.setAdMobDisplayer(AdMobDisplayer19(this@MiniAppDisplayActivity))
```

Comment:
Maybe we can remove this line.
Also, we can tell that `AdMobDisplayer` has been deprecated in changelog, users also can use `AdMobDisplayer19` instead.